### PR TITLE
Yamllint action

### PR DESCRIPTION
**Pull request by sra405**
What this does:
- searches through the `pathways` directory for yaml and lints
- fails or errors (in combo with forcing CI to pass this would stop errors being merged)
- add comments on the PR directed at the errors

Suggestions:
- Protect the `gh-pages` branch so only PRs can be committed
- Force CI to pass all checks prior to allowing merging

image

### CODE REVIEW

**Comment by sra405** [comment]
Ouch ... got burnt by my own automation 🔥

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: yamllint
+on:
```
[the remark this answers is not in the document]
Comment (sra405) [reply .github/workflows/lint.yml:3]: 😕  what

**Comment by grega** [comment]
Related: https://github.com/RaspberryPiFoundation/projects-admin/issues/1238

**Comment by sra405** [comment]
Approach sounds reasonable with ILT ([slack thread here](https://raspberrypifoundation.slack.com/archives/C02JBAA2NFP/p1666252713400549?thread_ts=1666204446.525399&cid=C02JBAA2NFP)). It might just need tweaking slightly to give the best workflow experience and ensure nothing broken goes into `gh-pages`

**Pull request merged**